app/services/lab/lims/utils.rb

```diff
@@ -79,10 +83,18 @@ def self.parse_date(str_date, fallback_date = nil)
       end
 
       def self.find_concept_by_name(name)
```

Comment:
```suggestion
def self.find_concept_by_name(name)
  concept_id = TEST_INDICATOR_MAPPINGS[name.upcase]
  query_param = concept_id.nil? ? {name: CGI.unescapeHTML(name)} : {concept_id: concept_id}
  
  ConceptName.joins(:concept).merge(Concept.all).where(query_param).first
end
```

Reply:
Refactored to use single line condition to build query condition